### PR TITLE
docs: update WP readme

### LINKED ISSUE #137: Move changelog into separate file
Per the [WordPress Plugin Handbook](https://developer.wordpress.org/plugins/wordpress-org/how-your-readme-txt-works/#file-size):

> While readmes are simple text files, having a file larger than 10k may result in errors. Your readme should be brief and to the point. The description should not be a sales pitch as much as a description of the plugin, what it does, and how to use it. Your install directions should be direct. Your FAQ should actually address issues.
> 
> As for your changelog, we recommend keeping the current release in the readme and splitting the rest out out into it’s own file — changelog.txt for example. By storing all the older changelog data there, you keep your readme small and allow the people who get really into long changelogs to read things on their own.
> 
> Similarly, if you need in-depth documentation with inline images and so on, direct people to your own website.

Our `readme.txt` currently weighs in at ~50 kB. We should split the changelog section into a separate file as suggested in the Handbook. This will also make #117 simpler.

### LINKED ISSUE #138: Required PHP/WP versions not displaying on upgrade
See https://github.com/markcheret/footnotes/discussions/104#discussioncomment-642453

### PR DESCRIPTION
**Pull request by Rumperuu**
Updates the WP readme (`readme.txt`) to follow the suggestions from the [WP Plugin Handbook](https://developer.wordpress.org/plugins/wordpress-org/how-your-readme-txt-works):

- split the changelog into a separate file (fix #137) and only keeping the last 5 items in the readme
- show the correct required versions (fix #138)
- add/remove/move readme sections
- add plugin summary and a few missing header items
- update screenshot to reflect new menu location (currently the SVN `assets/` dir is not implemented on this repo., so we'll just have to remember to replace `screenshot-1.png` with the attached photo when we release)

Tested with the [WP readme validator](https://wordpress.org/plugins/developers/readme-validator/) and no issues. I had to remove the ‘Installation’ section to get the FAQ to render properly on the [visual validator](https://wpreadme.com/) (see f6c630b commit message).

Once this PR is approved and merged, I'll generate another release, give it a quick check and (assuming nothing is broken) update the stable tag to 2.7.1 to push the new version out.

### CODE REVIEW

**Review by lolzim — APPROVED** [review]
Excellent, thanks!

File: footnotes.php [hunk 1]
```diff
@@ -2,12 +2,14 @@
 /**
  * Plugin Name: footnotes
  * Plugin URI: https://wordpress.org/plugins/footnotes/
- * Description: time to bring footnotes to your website! footnotes are known from offline publishing and everybody takes them for granted when reading a magazine.
+ * Description: footnotes lets you easily add highly-customisable footnotes on your WordPress Pages and Posts.
  * Author: Mark Cheret
+ * Author URI: https://cheret.de/
```
Comment (markcheret) [inline footnotes.php:7]: Please leve the URI as is.
https://cheret.org/footnotes
this page is specifically made for footnotes users. Everything else leads to confusion and people booking appointments with me, expecting they're free.

File: readme.txt [hunk 2]
```diff
@@ -1,714 +1,91 @@
 === footnotes ===
 Contributors: mark.cheret, lolzim, rumperuu, aricura, misfist, ericakfranz, milindmore22, westonruter, dartiss, derivationfr, docteurfitness, felipelavinz, martinneumannat, matkus2, meglio, spaceling, vonpiernik, pewgeuges
+Donate link: https://cheret.org/footnotes/
```
Comment (markcheret) [inline readme.txt:3]: The donation link is:
https://www.paypal.com/cgi-bin/webscr?cmd=_s-xclick&hosted_button_id=6Z6CZDW8PPBBJ

**Review by markcheret — CHANGES_REQUESTED** [review]
Looking good except for critical URLs...

File: readme.txt [hunk 3]
```diff
@@ -1,714 +1,91 @@
...
 Tags: footnote, footnotes, bibliography, formatting, notes, Post, posts, reference, referencing
 Requires at least: 3.9
 Tested up to: 5.7
 Requires PHP: 7.0
 Stable Tag: 2.7.0
```
Comment (markcheret) [inline readme.txt:8]: That needs to be bumped up to 2.7.1

File: readme.txt [hunk 4]
```diff
@@ -1,714 +1,91 @@
 === footnotes ===
 Contributors: mark.cheret, lolzim, rumperuu, aricura, misfist, ericakfranz, milindmore22, westonruter, dartiss, derivationfr, docteurfitness, felipelavinz, martinneumannat, matkus2, meglio, spaceling, vonpiernik, pewgeuges
+Donate link: https://cheret.org/footnotes/
 Tags: footnote, footnotes, bibliography, formatting, notes, Post, posts, reference, referencing
 Requires at least: 3.9
 Tested up to: 5.7
```
Comment (markcheret) [inline readme.txt:6]: Tested up to 5.7.1

**Review by markcheret — CHANGES_REQUESTED** [review]
Found 2 more things

**Comment by markcheret** [comment]
@Rumperuu do you want me to commit the required changes or are you happy to do them?
Once it's done, it can go out to WP SVN and I'd be delighted if you take the lead on that.

File: readme.txt (hunk 3 above)
Comment (Rumperuu) [reply readme.txt:8]: Once this is merged I'll update `trunk/`, give it a quick test to make sure nothing's gone horribly wrong (there shouldn't be any major change from the current `trunk/` but better safe than sorry) and then update the Stable Tag.

**Comment by Rumperuu** [comment]
> @Rumperuu do you want me to commit the required changes or are you happy to do them?
> Once it's done, it can go out to WP SVN and I'd be delighted if you take the lead on that.

I'm happy to handle it

**Review by markcheret — APPROVED** [review]
Looking good now. Thank you!

**Pull request merged**